### PR TITLE
Use only the values required for description

### LINKED ISSUE #2917: [Conditionals] Condition description renders incorrectly
1. Create a Condition Set with a condition that uses a 'between' criterion. Save it.
1. Note that the description will look correct:
Screen Shot 2020-04-10 at 4 35 43 PM
1. Edit the CS, then change the between condition to any other condition that uses a single input, like 'greater than'. Save it, then observe it uses the comma-based approach from between:
Screen Shot 2020-04-10 at 4 33 46 PM

### PR DESCRIPTION
**Pull request by shefalijoshi**
Resolves #2917

**Author Checklist**

- Changes address original issue? Yes
- Unit tests included and/or updated with changes? No
- Command line build passes? Yes
- Changes have been smoke-tested? Yes

### CODE REVIEW

File: src/plugins/condition/utils/operations.js [hunk 1]
```diff
@@ -34,6 +34,10 @@ const convertToStrings = (input) => {
     return stringInputs;
 };
 
+const joinValues = (values, length) => {
```
Comment (nikhilmandlik) [inline src/plugins/condition/utils/operations.js:37]: `const joinValues = (values = 1, length) => {` then can get rid of `1` in function call :P

Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js:37]: Yeah, but I want this to be explicitly defined. maybe the default should be the length of the values array. Let's leave it as is for now.

**Comment by akhenry** [comment]
## Author Checklist

* Changes address original issue? Y
* Unit tests included and/or updated with changes? N/A
* Command line build passes? Y
* Changes have been smoke-tested? Y

**Comment by shefalijoshi** [comment]
Testing notes: As described in #2917

**Pull request merged**